src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/VersionsCommand.java

```diff
@@ -97,6 +98,17 @@ public void onExecute(@Nonnull CommandSender sender, @Nonnull String[] args) {
 
             addJavaVersion(builder);
 
+            if (Slimefun.getUpdater().getBranch() == SlimefunBranch.STABLE) {
+                builder.append("\nYou are using a RC (stable) version, " +
+                        "\nDO NOT report any bugs to Discord/GitHub.\n")
+                    .color(ChatColor.RED)
+                    .event(new HoverEvent(HoverEvent.Action.SHOW_TEXT, new Text(
+                        "We do not provide support for stable versions, use with caution!\n" +
```

Comment:
Why `use with caution!` this is a stable build there should be no worries.
I get what you mean but i dont think this is the way to say it